enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/protocol/NettyPipelineBuilder.java

```diff
@@ -229,14 +230,14 @@ private void ensureErrorHandling()
             @Override
             public void exceptionCaught( ChannelHandlerContext ctx, Throwable cause )
             {
-                log.error( "Exception in inbound", cause );
+                log.error( format( "Exception in inbound for channel: %s", ctx.channel() ), cause );
                 ctx.channel().close();
             }
 
             @Override
             public void channelRead( ChannelHandlerContext ctx, Object msg )
             {
-                log.error( "Unhandled inbound message: " + msg );
+                log.error( format( "Unhandled inbound message: %s for channel: %s", msg, ctx.channel() ) );
```

Comment:
In this case we could use the logger's formatting overload instead of `String.format`